lib_nbgl/doc/nbgl_step.dox

```diff
@@ -0,0 +1,130 @@
+#ifdef NBGL_STEP
```

Comment:
Should it be called `nbgl_step_nanos.dox` to highlight the fact that it is for the nanos ONLY? 
The same question for `nbgl_flow.dox`.